tests/external_config/__init__.py

```diff
@@ -7,3 +7,91 @@
 # By accessing, using, copying or modifying this work you indicate your
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sys
+import os
+
+from mock import Mock
+
+# import the test base class
+test_python_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "python"))
```

Comment:
No need to add to sys.path for. The toolkit unit test framework does it for you.